.github/workflows/deployment.yaml

```diff
@@ -0,0 +1,104 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Deploy Bundle
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Version Check
+        id: check
+        uses: EndBug/version-check@v1.3.0
+
+      - name: Has version changed?
+        if: steps.check.outputs.changed != 'true'
+        run: 'echo "No version change :/" && exit 1'
+
+  build:
+    needs: check-version
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    
+    - run: npm ci
+    
+    - run: npm run build --if-present
+    
+    - run: npm test
+    
+    - name: Upload build files
+      uses: actions/upload-artifact@v2
+      with:
+        name: build-files
+        path: build
+
```

Comment:
As discussed, we are moving to `dist/` for kits, most kits don't have `build` so I am removing it here

```suggestion
```